build.gradle

```diff
@@ -110,7 +110,7 @@ dependencies {
     compile 'org.springframework.hateoas:spring-hateoas:0.19.0.RELEASE'
 
     providedCompile 'org.springframework.boot:spring-boot-starter-tomcat:1.3.2.RELEASE'
-    providedRuntime 'org.hsqldb:hsqldb:2.3.3'
+    providedRuntime 'org.hsqldb:hsqldb:2.3.1'
```

Comment:
Could be helpful to add a comment in order to explain that the version was downgraded and that version X should not be used for reason Y.